### PR TITLE
build: add long_description to setup

### PR DESCRIPTION
**Pull request by jansenk**
Setuptools was bumped and now we need this

### CODE REVIEW

**Comment by jansenk** [comment]
Not bumping version #, i'll just delete the tag and recreate

File: setup.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def load_requirements(*requirements_paths):
     version=VERSION,
     author='edX',
     description='An API for creating submissions and scores.',
+    long_description="An API for creating and scoring submissions for the open edX platform",
```
Comment (nsprenkle) [inline setup.py:55]: Nit: This is the correct branding: 

```suggestion
    long_description="An API for creating and scoring submissions for the Open edX platform",
```

**Review by nsprenkle — APPROVED** [review]
:shipit:

**Comment by jansenk** [comment]
ugh github suggestion auto-commit default commit messages aren't CC-compliant 🙄

**Pull request merged**